### DISCUSSION TITLE
[Script request] KeepassXC LXC

### DISCUSSION THREAD

**Discussion by FaboulousSan** [community-scripts/ProxmoxVE#445] [Request script]
### Application Name

KeepassXC 

### Website

https://github.com/linuxserver/docker-keepassxc

### Description

KeePassXC is a password management application that enables you to store, organize, and secure your credentials, passwords, and other sensitive information in a locally encrypted database. The [docker-keepassxc](https://github.com/linuxserver/docker-keepassxc) repository, maintained by LinuxServer.io, provides a Docker image of KeePassXC, ideal for quick deployment in a containerized environment. However, I would like to know if it is possible to make this application usable through an LXC container to take advantage of the specific benefits of this type of virtualization while retaining the features of KeePassXC.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by shrijayan** [comment]
What difference it is giving than vaultwarden?

**Comment by MickLesk** [comment]
Desktop Software Only. No Server / No Webfrontend.
